examples/package.json

```diff
@@ -0,0 +1,16 @@
+{
+  "name": "@whispir-node/recipes",
```

Comment:
To keep consistent with the `@whispir` scope, should we instead rename this to `@whispir/node-recipes`?